db/migrate/20220820131829_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Again, the reference name should be `author`. Please refer to the above similar comment for the other migration.

Reply:
Got it :+1: